File: ConsoleApp1/Minuser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1
{
    class Minuser : IProcess
    {
        public Minuser() { }
        public dynamic Process(dynamic A, dynamic B)
        {
            return A - B;
        }
        public int ModProcess(int A, int B, int M)
        {
            return (A - B) % M;
        }
    }
}

File: src/com/company/Minuser.java
package com.company;

public class Minuser implements Process {
    @Override
    public double process(double a, double b) {
        return a - b;
    }

    @Override
    public int modProcess(int a, int b, int m) {
        return (a - b) % m;
    }
}
